### PR TITLE
[Test Questions] Getting test questions from the back end

### PR DESCRIPTION
**Pull request by fnormand01**
# Description

This PR is introducing the action type/actions/reducer/initial state to get the test questions and update the redux state.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**How to use these new actions:**
image

**Then if we print the redux state called 'testQuestions':**
image

**We get that result:**
image

# Testing

Manual steps to reproduce this functionality:

- you can test these new actions using the code shown above in any desired file of the project. You'll just need to assure the redux connection with this selected file.

# Checklist

- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [x] My changes generate no new compiler warnings
- [ ] ~~My changes generate no new accessibility errors and/or warnings~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [ ] ~~My changes look good on IE 11+ and Chrome~~

### CODE REVIEW

File: frontend/src/modules/LoadTestContentRedux.js [hunk 1]
```diff
@@ -34,10 +35,28 @@ const getTestInstructions = testName => {
...
+    });
+    return await testQuestionsContent.json();
+  };
+};
+
 // Initial State
 const initialState = {
   testMetaData: "",
-  testInstructions: ""
+  testInstructions: "",
```
Comment (caleybrock) [inline frontend/src/modules/LoadTestContentRedux.js]: I think these should also start as objects.

Comment (fnormand01) [reply frontend/src/modules/LoadTestContentRedux.js]: Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/356/commits/501c8b119c0fbd7e1c4d1faa1867e83e8ddd4192

**Comment by fnormand01** [comment]
@caleybrock Postman tests have already been created to test these new APIs. These actions are only calling the API themselves. That means that there is anything else to test other than what's already tested with Postman.

**Comment by fnormand01** [comment]
@caleybrock I added basic tests related to _updateTestMetaDataState_ and _updateTestQuestionsState_ actions.
See commit #https://github.com/code-for-canada/project-thundercat/pull/356/commits/42f3e93271553edd901b1fcfacb00b0871c867ee and commit #https://github.com/code-for-canada/project-thundercat/pull/356/commits/19f505da05e9002b1a9c8d2180bec7fd559bcc82

**Pull request merged**